src/main/java/fr/esgi/al5_2/Tayarim/controllers/AdministrateurController.java

```diff
@@ -0,0 +1,173 @@
+package fr.esgi.al5_2.Tayarim.controllers;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.*;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.exceptions.*;
+import fr.esgi.al5_2.Tayarim.services.AdministrateurService;
+import fr.esgi.al5_2.Tayarim.services.AuthService;
+import fr.esgi.al5_2.Tayarim.services.ProprietaireService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/admin")
+public class AdministrateurController {
+
+  private final AdministrateurService administrateurService;
+  private final AuthService authService;
+
+  public AdministrateurController(AdministrateurService administrateurService,
+      AuthService authService) {
+    this.administrateurService = administrateurService;
+    this.authService = authService;
+  }
+
+    /*
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Block comment has incorrect indentation level 4, expected is 2, indentation should be the same level as line 41.